plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -58,11 +58,14 @@ const DEFAULT_CONFIG: GraphQLInstrumentationConfig = {
   mergeItems: false,
   depth: -1,
   allowValues: false,
+  ignoreFields: [],
 };
 
 const supportedVersions = ['>=14'];
 
 export class GraphQLInstrumentation extends InstrumentationBase {
+  private _ignoreMap!: Map<string, boolean>;
```

Comment:
The config option is called "ignoreFields".

How about calling this field "_ignoreFieldsMap" (here and in other places)? Stating "ignore" without any context what is ignored is less readable IMO.